app/grandchallenge/core/models.py

```diff
@@ -34,20 +34,6 @@ class Meta:
         abstract = True
 
 
-class UUIDModelNoAutoNow(models.Model):
```

Comment:
I already changed the base classes of the forum models back to `UUIDModel` when rebasing the feature branch. Didn't realize at that point that that change wouldn't be visible, but I hope that's ok.